build-image-src/Dockerfile-java8-al2

```diff
@@ -5,8 +5,8 @@ FROM amazonlinux:2
 COPY --from=0 / /rootfs
 
 # Installing by yum at copied location
-RUN yum groupinstall -y development --installroot=/rootfs && yum clean all
```

Comment:
why are we removing `yum clean all` ? is it because there is not any cache to clear?

Reply:
It's redundant as it's run L27 in the same layer.